src/elements/navigation/navigation-section/navigation-section.scss

```diff
@@ -54,19 +54,19 @@
   }
 }
 
-:host([data-state='opened']) {
+:host(:state(state-opened)) {
```

Comment:
navigation with section looks broken, but so far I don't see the reason
https://lyne-components-pr4222.app.sbb.ch/?path=/story/elements-sbb-navigation-sbb-navigation--with-navigation-section